merkletree/pad.go

```diff
@@ -121,11 +122,13 @@ func (pad *PAD) LatestSTR() *SignedTreeRoot {
 	return pad.latestSTR
 }
 
-func (pad *PAD) TB(name string, value []byte) (*TemporaryBinding, error) {
+func (pad *PAD) Sign(msg ...[]byte) []byte {
+	return pad.signKey.Sign(bytes.Join(msg, nil))
+}
+
+func (pad *PAD) NextIndex(name string) []byte {
```

Comment:
Yeah, I wasn't sure exactly where we landed on the discussion in https://github.com/coniks-sys/coniks-go/pull/105#discussion_r86062465

There seems to have been an agreement on "latest" being the most recently signed epoch.  Do we like "current" as the one we're presently accepting registrations for?  Or, as I used, "next"?

I suppose this prefix needs to apply to the signature as well.